libs/sdk-ui-filters/api/sdk-ui-filters.api.md

```diff
@@ -119,6 +121,8 @@ export type AttributeFilterControllerData = {
     parentFilterAttributes: IAttributeMetadataObject[];
     displayForms: IAttributeDisplayFormMetadataObject[];
     currentDisplayFormRef: ObjRef;
+    enableShowingFilteredElements?: boolean;
+    irrelevantSelection?: string[];
```

Comment:
Should this be `IAttributeElement[]` to make it consistent with the other selection sub states?